projects/env-semconv.md

```diff
@@ -0,0 +1,61 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructure and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics.
+
+Current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
+- TBD
+
+We want to have a unified metrics convention in order to better facilitate measuring sustainability metrics across different ecosystems
+
+### Deliverables
+
+- Semantic conventions for sustainability metrics
+- Reference Prototypes metrics
+- Recommended practices for reporting on sustainability metrics 
+
+### Staffing / Help Wanted
+
+**Project Leads**
+- @spazzy757
+
+**Sponsoring Members**
+- @jpkrohling 
```

Comment:
I confirmed that @pyohannes can be added here as a sponsoring member, thanks @pyohannes!